lib/Backends/OpenCL/OpenCL.h

```diff
@@ -183,6 +183,7 @@ class OCLBackend final : public Backend {
       case Kinded::Kind::ConvolutionNodeKind:
       case Kinded::Kind::DequantizeNodeKind:
       case Kinded::Kind::DivNodeKind:
+      case Kinded::Kind::FullyConnectedNodeKind:
```

Comment:
Why do we need this for the splat support?

Reply:
@ZchiPitt please make that change in a separate commit for clarity.